src/bsoncxx/types.hpp

```diff
@@ -530,6 +556,11 @@ struct BSONCXX_API b_codewscope {
         : code(std::forward<T>(code)), scope(std::forward<U>(scope)) {
     }
 
+    ///
+    /// Default constructor for b_codewscope.
+    ///
+    BSONCXX_INLINE b_codewscope() = default;
+
```

Comment:
Fixing the nit in a separate PR. Per our discussion we'll leave this fix in review so we have more time to think about it.